plugins/services/src/js/components/modals/NewCreateServiceModal.js

```diff
@@ -333,6 +325,26 @@ class NewServiceFormModal extends Component {
     return [];
   }
 
+  getResetState(nextProps = this.props) {
+    let newState = {
+      isJSONModeActive: false,
+      serviceConfig: this.getNormalizedServiceConfig(nextProps.service),
```

Comment:
Please use `getSpec` instead of `getNormalizedServiceConfig` as this method is only transforming pods and we should always operate on specs.